vllm/engine/arg_utils.py

```diff
@@ -209,7 +210,10 @@ def add_cli_args(
             default=EngineArgs.gpu_memory_utilization,
             help='the fraction of GPU memory to be used for '
             'the model executor, which can range from 0 to 1.'
-            'If unspecified, will use the default value of 0.9.')
+            'If unspecified, will use the default value of 0.9.'
+            'Recommend to set it to 0.8 for embedding models '
+            'to effectively handle memory demands while minimizing '
+            'the risk of OOM errors.')
```

Comment:
I feel this should go into documentation, not CLI help (it is specific only to embedding models)

Reply:
Removed the documentation. Embedding won't use this parameter.